src/main/java/com/sphy/hotelmanagementapplication/domain/Room.java

```diff
@@ -0,0 +1,124 @@
+package com.sphy.hotelmanagementapplication.domain;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity(name = "rooms")
+@DiscriminatorValue("rooms")
+public class Room {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "luxurity")
+    private int luxurity;
```

Comment:
Is this needed? Since the rooms exist only a hotel exists, maybe the stars system is enough

Reply:
είναι το πόσο ακριβό είναι το δωμάτιο και τι παρέχει που είπαμε πχ τζάκι τζακούζι κτλ για να έχουν και τα δωμάτια άλλη τιμή

Reply:
ok, thanks!